solutions/cses-1163.mdx

```diff
@@ -30,44 +30,44 @@ multiset.
 
 ```cpp
 #include <bits/stdc++.h>
-
 using namespace std;
 
-//constant initialization
-const int maxn=2e5+10;
-
-//variables used for the current problem
-int x,n,p; set<int> lights; multiset<int> dist;
-
-void solve() {
-	cin >> x >> n;
-	lights.insert(0); lights.insert(x);
-	dist.insert(x);
-	for (int i=0;i<n;++i){
-		cin >> p;
-		auto it1 = lights.upper_bound(p),it2 = it1; --it2;
-		dist.erase(dist.find(*it1-*it2));
-		dist.insert(p-*it2); dist.insert(*it1-p);
-		lights.insert(p);
-		auto ans = dist.end(); --ans;
+int main() {
+	std::ios::sync_with_stdio(false);
+	std::cin.tie(NULL);
+
+	int street_len;
+	int light_num;
+	cin >> street_len >> light_num;
+
+	set<int> lights{0, street_len};
+	multiset<int> dist{street_len};
+	for (int l = 0;l < light_num; l++) {
```

Comment:
```suggestion
	for (int l = 0; l < light_num; l++) {
```
omg my first time seeing kevin mess up spacing

Reply:
you saw nothing